org.adempiere.base/src/org/compiere/model/GridTab.java

```diff
@@ -2121,11 +2121,16 @@ public boolean hasAttachment()
 	 *	Get Attachment_ID for current record.
 	 *	@return AD_AttachmentID or 0 if no attachment.
 	 */
+	@SuppressWarnings("deprecation")

```

Comment:
if the method is still needed at so many places, I think we just shouldn't mark it as deprecated.

Reply:
Thanks @hengsin 
Committed a slightly different approach - more consistent with other points with same Record_ID+UU discovery